juju/framework.py

```diff
@@ -173,6 +174,10 @@ def __get__(self, obj, obj_type):
         return obj_type.__name__
 
 
+def _is_unbound_event(obj):
```

Comment:
This sounds like _is_event, but this one liner seems to obscure the logic more than it helps. We know need to go after its definition to understand the code.